pkg/tsdb/tempo/tempo_test.go

```diff
@@ -4,21 +4,15 @@ import (
 	"context"
 	"testing"
 
-	"github.com/grafana/grafana/pkg/infra/httpclient"
-	"github.com/grafana/grafana/pkg/models"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestTempo(t *testing.T) {
-	plug, err := New(httpclient.NewProvider())(&models.DataSource{})
-	executor := plug.(*tempoExecutor)
-	require.NoError(t, err)
-
-	t.Run("createRequest should set Auth header when basic auth is true ", func(t *testing.T) {
-		req, err := executor.createRequest(context.Background(), &models.DataSource{BasicAuth: true, BasicAuthUser: "john", BasicAuthPassword: "pass"}, "traceID")
+	t.Run("createRequest - success", func(t *testing.T) {
+		service := &Service{}
+		req, err := service.createRequest(context.Background(), &datasourceInfo{}, "traceID")
 		require.NoError(t, err)
-		assert.Equal(t, 2, len(req.Header))
-		assert.NotEqual(t, req.Header.Get("Authorization"), "")
```

Comment:
Nice work! LGTM, except for this small thing that is not equal, I think we might want it here after changes?

Reply:
the authorization is not set anymore in this method, not sure how to test it